content/departments/cloud/technical-docs/v2.0/onprem_data_migration.md

```diff
@@ -253,16 +257,16 @@ CREATE DATABASE "codeinsights-db";
 Ensure [databases have been reset](#reset-databases). Then, one by one, import each database from the bucket the customer has uploaded to:
 
 ```sh
-export TARGET_INSTANCE_PROJECT=$(mi2 instance get -jq '.status.gcp.projectId')
-export TARGET_INSTANCE_DB=$(mi2 instance get -jq '.status.gcp.cloudSQL[0].name')
-# from migration resources output
-export DB_DUMP_BUCKET="..."
+export TARGET_INSTANCE_PROJECT=$(mi2 instance get -jq '.status.gcp.projectId' | tr -d '"')
```

Comment:
I think you can also put this through `jq` again and get a raw value, or use `jq` the whole way through (`mi2 instances get | jq -raw  '.status.gcp.projectId'`)

Reply:
In practice though I'm surprised that this would cause issues, as we use similar mechanisms in Cloud automation 🤔